### PR TITLE
Bug report - GroupChat + MultimodalConversableAgent

### PR DESCRIPTION
**Pull request by joshkyh**
Suggest @BeibinLi as a reviewer, please. 

## Why are these changes needed?

A reproducible example that max_round of group chat do not limit the conversation length in GroupChat.

## Related issue number

image

## Checks

- [x ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by BeibinLi** [comment]
Thanks for finding this bug and contributing. 

I created an issue with GroupChat and MultimodalConversableAgent: https://github.com/microsoft/autogen/issues/628

File: test/agentchat/test_groupchat.py [hunk 1]
```diff
@@ -112,8 +122,60 @@ def test_plugin():
     assert len(groupchat.messages) == 2
 
 
+@pytest.mark.skipif(skip, reason="dependency is not installed")
+def test_group_chat_with_llm():
```
Comment (sonichi) [inline test/agentchat/test_groupchat.py:126]:
```suggestion
def test_group_chat_with_lmm():
```

File: test/agentchat/test_groupchat.py [hunk 2]
```diff
@@ -1,5 +1,15 @@
 import pytest
 import autogen
+import threading  # So that the function call runs for limited time
```
Comment (sonichi) [inline test/agentchat/test_groupchat.py:3]: where is this used?

Comment (joshkyh) [reply test/agentchat/test_groupchat.py]: I have removed threading -- Apologies as I was testing the feasibility of constraining the run-away group chat to 5 seconds but failed to do it with threading. It is not used anymore.

**Comment by BeibinLi** [comment]
@ joshkyh Thanks for updating the PR! I will fix the issue in about 12 hours, and then move your test into the "contrib" folder. Keep tuned, I will send you an invite after making the changes.

**Comment by joshkyh** [comment]
No worries, it's ok for me to close this PR too. No rush! Thanks for working on this. :)

**Comment by BeibinLi** [comment]
@joshkyh  See https://github.com/microsoft/autogen/issues/658 for more details. Closing this PR now, and we will create another PR to update the design accordingly. 

In the meanwhile, if you want a GroupChat flow with LMM models, you can contact me in Discord, I can give you a 2-line fix.

**Pull request closed without merging**